File: eBarberShop/eBarberShop.Model/Recenzije.cs
﻿namespace eBarberShop.Model
{
    public class Recenzije
    {
        public int RecenzijeId { get; set; }
        public string Sadrzaj { get; set; }
        public double Ocjena { get; set; }
        public DateTime DatumObjave { get; set; }
        public int KorisnikId { get; set; }
        public Korisnici Korisnik { get; set; }

    }
}

File: eBarberShop/UI/ebarbershop_mobile/lib/models/recenzije/recenzije.dart
import 'package:ebarbershop_mobile/models/korisnik/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';

part 'recenzije.g.dart';

@JsonSerializable()
class Recenzije {
  int recenzijeId;
  String sadrzaj;
  double ocjena;
  DateTime datumObjave;
  int korisnikId;
  Korisnik? korisnik;

  Recenzije(this.recenzijeId, this.sadrzaj, this.ocjena, this.datumObjave,
      this.korisnikId, this.korisnik);

  factory Recenzije.fromJson(Map<String, dynamic> json) =>
      _$RecenzijeFromJson(json);

  Map<String, dynamic> toJson() => _$RecenzijeToJson(this);
}
